tests/Core/ArcTable.Tests.fs

```diff
@@ -1623,6 +1623,46 @@ let private tests_AddColumns =
         ]
     ]
 
+
+//New Test for AddColumnFill
+let private tests_AddColumnFill = 
+    testList "AddColumnFill" [ 
+        testCase "addColumnFill to preexisting table with rows" (fun () ->  // AddColumnFill to Table with Rows // create ftestCase which prioritizes the testCase but only on local!!
+            let table = create_testTable()
+            let expectedValue = CompositeCell.createUnitizedFromString("1")
+            table.AddColumnFill(CompositeHeader.Parameter (OntologyAnnotation("chlamy_r")), expectedValue) //or .createTermfromString
+            Expect.equal table.ColumnCount 6 "ColumnCount"
+            Expect.equal table.RowCount 5 "RowCount"
+            let columnIndex = table.ColumnCount - 1
+            let getactualValue rowIndex  = table.Values.[columnIndex,rowIndex]
+            for ri in 0 .. table.RowCount - 1 do 
+                let value = getactualValue ri 
+                Expect.equal value expectedValue $"RowIndex: {ri}"
+        )
+        ftestCase "addColumnFill to empty table" (fun () -> // "Empty Table" fail if empty flag at least RowCount 1 if table empty etc
```

Comment:
Don't forget to switch from `ftestCase` to `testCase`

Reply:
> [!IMPORTANT]
> We should use `--fail-on-focused-tests` on our CI pipeline!
> 
> https://github.com/Freymaurer/Fable.Pyxpecto?tab=readme-ov-file#focused

Can you do this on your next commit @HLWeil ?

Reply:
We actually have multiple options how we want to tackle this:
1. Pass any `<env_variable>` and use `#if <env_variable>` syntax to add this argument in pyxpecto run tests call
2.  propagate --fail-on-focused to pyxpecto and edit workflow